### DISCUSSION TITLE
New Install on Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by efernbach** [xournalpp/xournalpp#5838] [General]
I installed from the App center, after downloading the .deb file didn't work. The app runs now but I have two problems:
1. using a wacom tablet, I have to open and close preferences before the stylus works.
2. when I set the pen preferences and paper background these settings are gone when I start a new document. After intensive research I couldn't find a way to safe those preferences.

**Comment by GithubAnon0000** [comment]
> installed from the App center

If ubuntu uses an older version than [v1.2.3](https://github.com/xournalpp/xournalpp/releases/tag/v1.2.3), please upgrade and see if your issues persist. 

[if issues are still there]

> using a wacom tablet, I have to open and close preferences before the stylus works.

If this had not been reported yet you can build the latest version of xournalpp with [input debugging enabled](https://github.com/xournalpp/xournalpp/wiki/How-to-provide-debug-info-for-input-related-bugs) and open a new issue.

> when I set the pen preferences [...] these settings are gone when I start a new document.

What preferences? You mean those from `edit > preferences > defaults`?

> when I set the [...] paper background these settings are gone when I start a new document.

You might want `journal > configure page template`.
